server/routes/auth.js

```diff
@@ -4,17 +4,23 @@ import AuthController from '../controllers/AuthController';
 import middlewares from '../middlewares';
 
 const { verifyToken, UserValidator } = middlewares;
-
 const auth = express.Router();
 const BASE_URL = '/auth';
-const { createUser, login, verifyUser } = AuthController;
-const { signUpValidation, loginValidation } = UserValidator;
-
+const {
+  createUser, login, verifyUser, resetpasswordEmail, resetPassword
+} = AuthController;
+const {
+  signUpValidation, loginValidation, PasswordValidation, EmailValidation
+} = UserValidator;
 // Sign up route
 auth.post(`${BASE_URL}/signup`, signUpValidation(), createUser);
-// Login route
+/* Login route */
 auth.post(`${BASE_URL}/login`, loginValidation(), login);
-// Verify User route
+/* Verify User route */
 auth.get(`${BASE_URL}/verify`, verifyToken, verifyUser);
+/* route responsible for sending password reset link to owners mail */ 
+auth.get(`${BASE_URL}/passwordreset`, EmailValidation(), resetpasswordEmail);
```

Comment:
Why are you using a get request here? Why is it not a post request.

Reply:
> Why are you using a get request here? Why is it not a post request.
> 
> because you are requesting for an access to change your password which would be sent to your mail,

Yes. But you added a body to the request. GET requests generally don't have a body with it.